web_src/js/utils.js

```diff
@@ -26,13 +26,8 @@ export function isObject(obj) {
 
 // returns whether a dark theme is enabled
 export function isDarkTheme() {
-  if (document.documentElement.classList.contains('theme-auto')) {
-    return window.matchMedia('(prefers-color-scheme: dark)').matches;
-  }
-  if (document.documentElement.classList.contains('theme-arc-green')) {
-    return true;
-  }
-  return false;
+  const style = window.getComputedStyle(document.documentElement);
+  return style.getPropertyValue('--is-dark-theme').trim().toLowerCase() === 'true';
```

Comment:
What is the behavior of `getPropertyValue` when the variable is not set?
The empty string?

Reply:
Then ignore this comment.
I would resolve it, but for whatever reason, GitHub does not allow to resolve your own comments.

Reply:
Yes, it is.

```js
window.getComputedStyle(document.documentElement).getPropertyValue('--is-dark-theme')
''
```